tests/test_VideoFileClip.py

```diff
@@ -44,9 +44,9 @@ def test_ffmpeg_resizing():
 
 
 def test_copied_videofileclip_write_videofile(util):
-    """Check that a copied ``VideoFileClip`` can be renderizable using
-    ``write_videofile``, opened from that render and the new video shares
-    the same data that the original clip.
+    """Check that a copied ``VideoFileClip`` can be rendered using
+    ``write_videofile``, open that rendered file and check that the
```

Comment:
I think the original meant to say: "Check that... [these three things can be done]", so I wouldn't change "(can be) opened" to an action.

Maybe "Check that (...) can be rendered (...), opened once rendered, and that the file contains the same data (...)"